test/unittests/evm_calls_test.cpp

```diff
@@ -484,6 +513,7 @@ TEST_F(evm_calls, call_with_value_low_gas)
 
 TEST_F(evm_calls, call_oog_after_balance_check)
 {
+    host.accounts[{}] = {};
```

Comment:
What is this, account with address 0?

Reply:
Ah ok, I see the explanation in the commit message.

Maybe `0x0000000000000000000000000000000000000000_address` would be more cleare